src/marks/tick.js

```diff
@@ -61,13 +62,13 @@ export class TickX extends AbstractTick {
   _x2(scales, {x: X}) {
     return i => X[i];
   }
-  _y1(scales, {y: Y}, {marginTop}) {
+  _y1({y}, {y: Y}, {marginTop}) {
     const {insetTop} = this;
-    return Y ? i => Y[i] + insetTop : marginTop + insetTop;
+    return Y && !isCollapsed(y) ? i => Y[i] + insetTop : marginTop + insetTop;
   }
   _y2({y}, {y: Y}, {height, marginBottom}) {
     const {insetBottom} = this;
-    return Y ? i => Y[i] + y.bandwidth() - insetBottom : height - marginBottom - insetBottom;
+    return Y  && !isCollapsed(y) ? i => Y[i] + y.bandwidth() - insetBottom : height - marginBottom - insetBottom;
```

Comment:
Okay, next question: do we need to use isCollapsed here? Or should we just check if the scale is defined?

```suggestion
    return Y && y ? i => Y[i] + y.bandwidth() - insetBottom : height - marginBottom - insetBottom;
```

Reply:
It can be defined by another mark
```
  Plot.tickX([1, 1.1, 2, 3], Plot.dodgeY()),
  Plot.dotY([1, 2, 3, 4])
```

... in which case it's not enough to check collapsed, we should also check if it has a bandwidth method (next commit).